crates/types/src/consensus.rs

```diff
@@ -299,7 +299,27 @@ impl<TYPES: NodeType> Consensus<TYPES> {
     }
 
     /// Update the validated state map with a new view_number/view combo.
+    ///
+    /// Skips the update if the new view contains less information than the exisiting view with the
```

Comment:
What do you think about this returning a result or something instead of just a blank return, and the function could perhaps be renamed to something like `try_update...` to signify the fallible operation? The thing about it right now is that when it's called, it might fail, and the caller may have no idea that it failed, which can make debugging harder.

Reply:
Yeah how about this -- I can make it similar to `update_high_qc`, i.e., using `ensure` and `Result` to let the caller know something is wrong.

Reply:
Perfect